Intersect (Core)/Enums/VariableType.cs

```diff
@@ -6,12 +9,20 @@ namespace Intersect.Enums
 
     public enum VariableType
     {
+        [Description("Player Variable")]
```

Comment:
How does this description get translated? If it doesn't, why do we need this attribute?

Reply:
it doesn't - I forgot about this and need some help in getting these descriptions translated :P.

I know you pointed me to a place where you did something similar (translatable attribute) but TBH I was very confused over there.

Reply:
![image](https://github.com/AscensionGameDev/Intersect-Engine/assets/1476550/1dd1267f-78bc-4702-8f16-b9376d4edf6a)
The attributes point to a section in the editor's strings class (which is why they are just strings instead of using `nameof()` since the attributes are  used in the Core lib).

You can look at the editor's `Localizer` for more info.

I would also say that for now rather than adding new attributes, you should be using the existing `Editor*Attribute`s, referring to how I used them on map attributes in conjunction with the editor's localizer.